rest-api/dao/physical_measurements_dao.py

```diff
@@ -215,19 +216,24 @@ def set_measurement_ids(physical_measurements):
         measurement_count += 1
 
   def insert_with_session(self, session, obj):
+    is_amendment = False
     if obj.logPosition is not None:
       raise BadRequest('%s.logPosition must be auto-generated.' % self.model_type.__name__)
     obj.logPosition = LogPosition()
     obj.final = True
     obj.created = clock.CLOCK.now()
     resource_json = json.loads(obj.resource)
     for extension in resource_json['entry'][0]['resource'].get('extension', []):
-      url = extension.get('url', '')
-      if url != _AMENDMENT_URL:
-        logging.info('Ignoring unsupported extension for PhysicalMeasurements: %r.' % url)
```

Comment:
So previously were we logging "Ignoring unsupported extension" for the created / finalized extensions?